kitsune/sumo/static/sumo/scss/form-wizard-configuration-step.styles.scss

```diff
@@ -0,0 +1,55 @@
+#header,
+#sync-status-container {
+  display: flex;
+  align-items: center;
+}
+
+#header {
+  padding-block-end: 12px;
+  border-bottom: 1px solid var(--color-light-gray-03);
+}
+
+.icon {
+  height: 16px;
+  margin-inline-end: 12px;
+}
+
+#header > a {
+  margin-inline-start: auto;
+}
+
+#sync-status {
+  font-size: 10px;
+  font-weight: 700;
+  color: var(--color-white);
+  border-radius: 34px;
+  padding: 2px 16px;
+  margin-inline-start: 12px;
+
+  /**
+   * TODO: Should this be in the color swatches somewhere?
+   */
+  background-color: #E22850;
```

Comment:
Seems like that's a Protocol color but the Protocol colors aren't available 🤔

I wonder if we should go with --color-warning: https://github.com/mozilla/kitsune/blob/main/kitsune/sumo/static/sumo/scss/config/_project-theme.scss#L12

Otherwise we can look into bringing in the Protocol colors for the form steps.

Reply:
`--color-error` is probably closer to the red that Tati was looking for. I'll go with that, and file a follow-up bug for trying to bring in some more colours from Protocol.